scripts/galera_new_cluster.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+
+VERSION="@VERSION@@MYSQL_SERVER_SUFFIX@"
+COMPILATION_COMMENT="@COMPILATION_COMMENT@"
+
+/usr/bin/systemctl set-environment MYSQLD_OPTIONS='--wsrep-new-cluster' && \
+     /usr/bin/systemctl start mariadb.service
+
+/usr/bin/systemctl set-environment MYSQLD_OPTIONS=''
```

Comment:
Just some thoughts. Even though the right place to list mysqld options is my.cnf, wasn't MYSQLD_OPTIONS supposed to be used by admins? If so, it's quite unexpected that this script sets and then resets it, without keeping original value.